linera-base/src/crypto.rs

```diff
@@ -172,6 +180,38 @@ impl Serialize for CryptoHash {
     }
 }
 
+impl Serialize for CryptoHashVec {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::ser::Serializer,
+    {
+        // if serializer.is_human_readable() {
+        //     let s = self
+        //         .0
+        //         .iter()
+        //         .map(|hash| hash.to_string())
+        //         .collect::<Vec<_>>();
+        //     serializer.collect_seq(s)
+        // } else {
+        serializer.serialize_newtype_struct("CryptoHashVec", &self.0)
+        // }
```

Comment:
TODO: Should be safe to just have the non-human serialization. We don't expose the vector anywhere, it's a temporary value used internally.